inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sort/SortFlinkConfigOperator.java

```diff
@@ -318,8 +318,10 @@ private void addAuditId(Map<String, Object> properties, String type, IndicatorTy
                         }
                         throw new BusinessException("current audit id can not find cdc audit information");
                     }).collect(Collectors.toList());
-            properties.putIfAbsent("metrics.changelog.audit.key",
-                    Joiner.on(InlongConstants.AMPERSAND).join(cdcAuditIdList));
+            if (CollectionUtils.isNotEmpty(cdcAuditIdList)) {
+                properties.putIfAbsent("metrics.changelog.audit.key",
```

Comment:
"metrics.changelog.audit.key" 
better to use a constant

Reply:
Done.